bitnami/harbor/requirements.yaml

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
-  version: 4.x.x
+  version: 5.x.x
```

Comment:
A new PostgreSQL major is coming, it should be merged today: https://github.com/helm/charts/pull/15643